examples/angular/tools/ngsw_config.bzl

```diff
@@ -11,27 +11,25 @@ def ngsw_config(name, config, index_html, src, out = None, **kwargs):
 
     nodejs_binary(
         name = ngsw_config_name,
-        data = ["@npm//@angular/service-worker", index_html, config, src],
+        data = ["@npm//@angular/service-worker"],
         visibility = ["//visibility:private"],
         entry_point = "@npm//:node_modules/@angular/service-worker/ngsw-config.js",
     )
 
     cmd = """
 mkdir -p $@
 cp -R $(locations {TMPL_src})/. $@/
-cp $(location {TMPL_index}) $@/index.html
```

Comment:
This index file is already in `TMPL_src` so this was causing a "cannot overwrite readonly file" error on ubuntu. I believe it's the exact same file so this was unnecessary (or maybe pre-rendering previously required it 🤷).